stock_reserve/__openerp__.py

```diff
@@ -32,6 +32,7 @@
              ],
  'demo': [],
  'data': ['view/stock_reserve.xml',
+          'view/stock_location.xml',
```

Comment:
remove empty key
version should change as well: not sure 8.0.0.2.1 or 8.0.0.3.0